concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -106,6 +106,7 @@ migratePersistentBakerInfoEx ::
     t m (PersistentBakerInfoEx (AccountVersionFor pv))
 migratePersistentBakerInfoEx StateMigrationParametersTrivial = migrateReference return
 migratePersistentBakerInfoEx StateMigrationParametersP5ToP6{} = migrateReference return
+migratePersistentBakerInfoEx StateMigrationParametersP6ToP7 = migrateReference return
```

Comment:
When pattern-matching, I would tend to use `StateMigrationParametersP6ToP7{}` as it will be unchanged if additional parameters are added later. The parameters generally only affect specific parts of the migration, so I don't think there's a benefit in the compiler forcing you to revise every pattern match if/when parameters are added.